nav2_waypoint_follower/src/waypoint_follower.cpp

```diff
@@ -35,7 +35,7 @@ WaypointFollower::WaypointFollower()
   declare_parameter("loop_rate", 20);
   nav2_util::declare_parameter_if_not_declared(
     this, std::string("waypoint_task_executor_plugin"),
-    rclcpp::ParameterValue(std::string("waypoint_task_executor")));
+    rclcpp::ParameterValue(std::string("photo_at_waypoint")));
```

Comment:
don't change this default, I think the wait plugin is more general. Lets leave it the waiting plugin (with the updated name you changed it to)